templates/user/auth/signup_inner.tmpl

```diff
@@ -54,23 +54,27 @@
 				{{end}}
 			{{end}}
 
-			{{if .OAuth2Providers}}
 			<div class="divider divider-text">
 				{{ctx.Locale.Tr "sign_in_or"}}
 			</div>
 			<div id="oauth2-login-navigator" class="tw-py-1">
 				<div class="tw-flex tw-flex-col tw-justify-center">
 					<div id="oauth2-login-navigator-inner" class="tw-flex tw-flex-col tw-flex-wrap tw-items-center tw-gap-2">
+					{{if .OAuth2Providers}}
 						{{range $provider := .OAuth2Providers}}
 							<a class="{{$provider.Name}} ui button tw-flex tw-items-center tw-justify-center tw-py-2 tw-w-full oauth-login-link" href="{{AppSubUrl}}/user/oauth2/{{$provider.DisplayName}}">
 								{{$provider.IconHTML 28}}
 								{{ctx.Locale.Tr "sign_in_with_provider" $provider.DisplayName}}
 							</a>
 						{{end}}
+					{{end}}
+					<a class="openid ui button tw-flex tw-items-center tw-justify-center tw-py-3 tw-w-full" href="{{AppSubUrl}}/user/login/openid">
+						{{svg "fontawesome-openid" 28}}&nbsp;
+						{{ctx.Locale.Tr "sign_in_with_provider" "OpenID"}}
+					</a>
```

Comment:
Does this render unconditionally on every login form? I think some condtion is needed because a default instance ought to not have openid enabled, or does it?

Reply:
Missed the conditional render within the navbar template. Have addressed this and moved all alt login render logic to a new template.

Reply:
@silverwind can I resolve this if it's been addressed to your satisfaction?